### PR TITLE
[omevv_baseline_profile_info] - Fetch baseline profile information

### PR DESCRIPTION
**Pull request by Kritika-Bhateja-03**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: playbooks/omevv/omevv_baseline_profile_info.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+- name: Dell OpenManage Ansible device inventory details.
+  hosts: omevv
+  gather_facts: false
+  tasks:
+    - name: Retrieve all baseline profile information.
+      dellemc.openmanage.omevv_baseline_profile_info:
+        hostname: "{{ hostname }}"
+        vcenter_uuid: "{{ vcenter_uuid }}"
```
Comment (sachin-apa) [inline playbooks/omevv/omevv_baseline_profile_info.yml:22]: Good to add a task on how do we fetch the venter uuid and pass to the module

**Review by ShivamSh3 — DISMISSED** [review]

File: playbooks/omevv/omevv_baseline_profile_info.yml (hunk 1 above)
Comment (Kritika-Bhateja-03) [reply playbooks/omevv/omevv_baseline_profile_info.yml:22]: Done

Comment (ABHISHEK-SINHA10) [reply playbooks/omevv/omevv_baseline_profile_info.yml:22]: Need to use register in first task and then use that for vcenter_uuid.

File: plugins/module_utils/omevv_utils/omevv_firmware_utils.py
Comment (ABHISHEK-SINHA10) [inline plugins/module_utils/omevv_utils/omevv_firmware_utils.py:1]: Please compare the file from Saksham's PR there's lot of changes in return value.

Comment (ABHISHEK-SINHA10) [reply plugins/module_utils/omevv_utils/omevv_firmware_utils.py:1]: there'll be merge conflict

Comment (Kritika-Bhateja-03) [reply plugins/module_utils/omevv_utils/omevv_firmware_utils.py:1]: Updated

File: playbooks/omevv/omevv_baseline_profile_info.yml (hunk 1 above)
Comment (Kritika-Bhateja-03) [reply playbooks/omevv/omevv_baseline_profile_info.yml:22]: Updated

**Review by sachin-apa — DISMISSED** [review]

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Pull request merged**